dashboard/src/actions/charts.ts

```diff
@@ -57,49 +57,62 @@ const allActions = [
 ].map(getReturnOfExpression);
 export type ChartsAction = typeof allActions[number];
 
+async function httpGet(dispatch: Dispatch<IStoreState>, targetURL: string) {
+  try {
+    const response = await fetch(targetURL);
+    const json = await response.json();
+    if (!response.ok) {
+      const error = json.data || response.statusText;
+      if (response.status === 404) {
+        dispatch(errorChart(new MissingChart(error)));
```

Comment:
I have mixed feelings here, I am not sure if doing all this specific code makes sense. I mean, 404 means the same for all kind of resources.

Reply:
Agreed, we should just raise the NotFoundError here